corehq/motech/openmrs/static/openmrs/js/openmrs_importers.js

```diff
@@ -23,6 +23,7 @@ hqDefine('openmrs/js/openmrs_importers', [
         // using the Django form field names.
         self.username = ko.observable(properties["username"]);
         self.password = ko.observable(properties["password"]);
+        self.notify_email = ko.observable(properties["notify_email"]);
```

Comment:
I think that the new standard is to use hungarian notation https://github.com/dimagi/commcare-hq/pull/25394/files